src/chapters/booleans-and-conditionals/equality.rst

```diff
@@ -0,0 +1,118 @@
+.. _equality:
+
+Equality
+========
+
+Loose Equality With ``==``
+--------------------------
+
+.. index:: equality, ==
+
+In the section :ref:`booleans`, we learned about the comparison operators ``==`` and ``!=``, which test whether two values are equal or not equal, respectively. We also tipped you off to some quirks of the ``==`` operator. Specifically, when its operands are of different types, then the returned boolean value may not be what you expect.
```

Comment:
Grammar: 'We' in the first sentence implies a role of learners, but the 'we' in the second sentence implies a teacher role.  

To avoid this conflict, we (teachers) should remove the 'we' from the second sentence.  :-) 'We' could try something like, "However, there are some quirks with using the ``==`` operator."

The third sentence can return to the we (learners) role: "These often occur when we use ``==`` to compare different data types."

Reply:
Fixed.